apps/aot/src/app/events/(leaderboard)/_components/stage.tsx

```diff
@@ -1,3 +1,5 @@
+// SVG props don't play well with react/no-unknown-property.
+/* eslint-disable react/no-unknown-property */
```

Comment:
🤔 this might be a bug in the rule? I don't know, I don't typically use it because TypeScript does a better version of this check.

[Question] Is there any reason to keep [`react/no-unknown-property`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md)?

Reply:
we can remove that rule. im not very familiar with it tbh